src/pkg/selectors/selectors_test.go

```diff
@@ -239,3 +240,126 @@ func (suite *SelectorSuite) TestSplitByResourceOnwer() {
 		})
 	}
 }
+
+// TestPathCategoriesAll includes all the included selectors do not return `Unknown` Category
+func (suite *SelectorSuite) TestPathCategoriesAll() {
+	users := []string{tester.LoadTestM365UserID(suite.T())}
+
+	table := []struct {
+		name        string
+		getSelector func(t *testing.T) *Selector
+		isErr       assert.ErrorAssertionFunc
+	}{
+		{
+			name:  "empty",
+			isErr: assert.Error,
+			getSelector: func(t *testing.T) *Selector {
+				return &Selector{}
+			},
+		},
+		{
+			name:  "Mail_B",
+			isErr: assert.NoError,
+			getSelector: func(t *testing.T) *Selector {
+				sel := NewExchangeBackup(users)
+				sel.Include(sel.MailFolders([]string{"MailFolder"}, PrefixMatch()))
+				return &sel.Selector
+			},
+		},
+		{
+			name:  "Mail_R",
+			isErr: assert.NoError,
+			getSelector: func(t *testing.T) *Selector {
+				sel := NewExchangeRestore(users)
+				sel.Include(sel.MailFolders([]string{"MailFolder"}, PrefixMatch()))
+				return &sel.Selector
+			},
+		},
+		{
+			name:  "Contacts",
+			isErr: assert.NoError,
+			getSelector: func(t *testing.T) *Selector {
+				sel := NewExchangeBackup(users)
+				sel.Include(sel.ContactFolders([]string{"Contact Folder"}, PrefixMatch()))
+				return &sel.Selector
+			},
+		},
+		{
+			name:  "Contacts_R",
+			isErr: assert.NoError,
+			getSelector: func(t *testing.T) *Selector {
+				sel := NewExchangeRestore(users)
+				sel.Include(sel.ContactFolders([]string{"Contact Folder"}, PrefixMatch()))
+				return &sel.Selector
+			},
+		},
+		{
+			name:  "Events",
+			isErr: assert.NoError,
+			getSelector: func(t *testing.T) *Selector {
+				sel := NewExchangeBackup(users)
+				sel.Include(sel.EventCalendars([]string{"July"}, PrefixMatch()))
+				return &sel.Selector
+			},
+		},
+		{
+			name:  "Events_R",
+			isErr: assert.NoError,
+			getSelector: func(t *testing.T) *Selector {
+				sel := NewExchangeRestore(users)
+				sel.Include(sel.EventCalendars([]string{"July"}, PrefixMatch()))
+				return &sel.Selector
+			},
+		},
+		{
+			name:  "SharePoint Pages",
+			isErr: assert.NoError,
+			getSelector: func(t *testing.T) *Selector {
+				sel := NewSharePointBackup(users)
+				sel.Include(sel.Pages([]string{"Something"}, SuffixMatch()))
```

Comment:
Any particular reason to use suffix match here?

Reply:
None